r1cs-std/Cargo.toml

```diff
@@ -36,13 +36,12 @@ algebra = { path = "../algebra", default-features = false, features = [ "bls12_3
 [features]
 default = ["std"]
 full = [ 
-  "bls12_377", "ed_on_bn254", "ed_on_bls12_381", "ed_on_bls12_377", "ed_on_cp6_782", 
```

Comment:
Curious - why remove baby jubjub?

Reply:
Hmm that seems to be an oversight, it looks like it disappeared in some rebasing. I'll add it back, thanks for noticing =)

Reply:
OK, fixed in [`8325150` (#186)](https://github.com/scipr-lab/zexe/pull/186/commits/832515095adb93aba95a018c62962e1ef2891521)